setup-nexus-go/action.yml

```diff
@@ -0,0 +1,20 @@
+name: setup-nexus-go
+inputs:
+  endpoint:
+    required: true
+  login:
+    required: true
+  password:
+    required: true
+runs:
+  using: composite
+  steps:
+    - run: |
+        echo GONOSUMDB=github.com/sportsball-ai >> $GITHUB_ENV
+        echo GOPROXY=https://${{ inputs.endpoint }}/repository/go >> $GITHUB_ENV
+        echo GOSUMDB="sum.golang.org https://${{ inputs.endpoint }}/repository/gosumdb" >> $GITHUB_ENV
+        rm -f ~/.netrc
+        echo machine ${{ inputs.endpoint }} >> ~/.netrc
```

Comment:
Instead of being multiple lines of echo, can we just use a bash heredoc?
https://mywiki.wooledge.org/HereDocument